src/ontology/uberon-edit.obo

```diff
@@ -224751,6 +224759,14 @@ is_a: UBERON:0002553 ! anatomical cavity
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-07-05T11:30:55Z xsd:dateTime
 
+[Term]
+id: UBERON:8480023
+name: intralobular stroma of mammary gland
```

Comment:
"In contrast, the functional lobular units of the human mammary gland are surrounded by loose intralobular connective tissue, consisting primarily of fibroblasts" from the reference. 
Could you add the axiom: 
'composed primarily of' some 'fibroblast'